tests/phpunit/tests/widgets.php

```diff
@@ -686,6 +686,53 @@ public function test_wp_widget_get_settings() {
 		$this->assertArrayNotHasKey( 0, $never_used );
 	}
 
+	/**
+	 * @covers WP_Widget::get_settings
+	 * @ticket 54677
+	 */
+	public function test_wp_widget_initiates_widget_with_alt_option() {
+		/*
+		 * Pretend the recent posts widget is new.
+		 *
+		 * The widget contains an alt (legacy) option so both the
+		 * new and the old option need to be deleted.
+		 */
+		delete_option( 'widget_recent-posts' );
+		delete_option( 'widget_recent_entries' );
+
+		wp_widgets_init();
+		$this->assertSameSetsWithIndex( array( '_multiwidget' => 1 ), get_option( 'widget_recent-posts' ) );
+		$this->assertFalse( get_option( 'widget_recent_entries' ) );
```

Comment:
Per [Writing PHPUnit Tests - Using Assertions](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions), test methods with multiple assertions must have a `message` argument to explain the failure. This also applies to the other test method.